internal/provision/devices.go

```diff
@@ -70,5 +70,13 @@ func createDevice(dc common.DeviceConfig) error {
 	device.Id = id
 	cache.Devices().Add(*device)
 
+	err = common.Driver.AddDevice(device.Name, device.Protocols, device.AdminState)
```

Comment:
When we provision devices from configuration, they're added to Core Metadata right? And if so, won't that generate a callback for the new device?  Just wondering why we're calling the Driver.AddDevice in multiple places.

Reply:
@tonyespy you are right.  this is a historical issue.
if you remember, we didn't support Add Device callback from metadata, only update adminState callback supported.  At that time, I had to add the pre-defined and existing Devices to cache in provision package.  Now, we can remove it.

Could you please merge this one first and I will remove them in issue #317?
Thanks

Reply:
@cloudxxx8 please either open a PR for #317 or remove the code in this PR, and I'll approve this PR.

Reply:
@tonyespy I've removed the lines